### PR TITLE
Solve missing clip_sample implementation in FlaxDDIMScheduler.

### PR DESCRIPTION
**Pull request by hi-sushanta**
Hi, In this pull request add a doc-string with two required parameters to solve this problem and also work some operations to solve this problem.  

The issue looks like that: #6901 
## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: src/diffusers/schedulers/scheduling_ddim_flax.py [hunk 1]
```diff
@@ -117,6 +119,8 @@ def __init__(
         beta_end: float = 0.02,
         beta_schedule: str = "linear",
         trained_betas: Optional[jnp.ndarray] = None,
+        clip_sample: bool = True,
+        clip_sample_range: float = 1.0,
         set_alpha_to_one: bool = True,
```
Comment (sayakpaul) [inline src/diffusers/schedulers/scheduling_ddim_flax.py:124]: Why are these two being added here?

**Review by sayakpaul — COMMENTED** [review]
@pcuenca WDYT?

**Comment by sayakpaul** [comment]
The commit history of this is borked. Would be nice to fix it either by force-pushing or by opening a new PR.

**Review by pcuenca — COMMENTED** [review]
Looks good to me. Could you please address @sayakpaul's comments about the commit history?

File: src/diffusers/schedulers/scheduling_ddim_flax.py [hunk 2]
```diff
@@ -266,6 +270,12 @@ def step(
                 f"prediction_type given as {self.config.prediction_type} must be one of `epsilon`, `sample`, or"
                 " `v_prediction`"
             )
+
```
Comment (pcuenca) [inline src/diffusers/schedulers/scheduling_ddim_flax.py:273]:
```suggestion

```

Comment (pcuenca) [inline src/diffusers/schedulers/scheduling_ddim_flax.py:273]:
```suggestion

```

Comment (pcuenca) [inline src/diffusers/schedulers/scheduling_ddim_flax.py:273]:
```suggestion

```

File: src/diffusers/schedulers/scheduling_ddim_flax.py [hunk 3]
```diff
@@ -266,6 +270,11 @@ def step(
                 f"prediction_type given as {self.config.prediction_type} must be one of `epsilon`, `sample`, or"
                 " `v_prediction`"
             )
+        # 4. Clip or threshold "predicted x_0"
```
Comment (pcuenca) [inline src/diffusers/schedulers/scheduling_ddim_flax.py:273]:
```suggestion

        # 4. Clip or threshold "predicted x_0"
```

**Comment by pcuenca** [comment]
The commit history is strange but the diff is ok, I think it's save to squash. Thanks, @hi-sushanta!

**Pull request merged**